### PR TITLE
Add option to change 'tabSize' in settings panel

### LINKED ISSUE #274: Tab Size(Number of spaces for tab) in settings.

### LINKED ISSUE #298: Add option to change 'tabSize' in settings panel
- Add option/functionality to change tab-size (number of spaces) in the editor's setting panel.

### PR DESCRIPTION
**Pull request by dakshkhetan**
Closes #298 
Closes #274 

GIF for reference:

ScreenRecord

### CODE REVIEW

**Comment by dakshkhetan** [comment]
@hereisnaman Please review. 😀
Thanks!

File: src/components/editor/Settings.vue [hunk 1]
```diff
@@ -39,6 +45,7 @@
         },
         fontOptions: ['Lucida Console', 'Anonymous Pro', 'Courier', 'Droid Sans Mono', 'Inconsolata', 'Source Code Pro', 'Ubuntu Mono'],
         sizeOptions: Array(30).fill(0).map((el, ind) => 6 + (2 * ind)),
+        tabSizeOptions: Array(4).fill(0).map((el, ind) => 2 + (2 * ind))
```
Comment (hereisnaman) [inline src/components/editor/Settings.vue:48]: Don't need 6.

**Comment by hereisnaman** [comment]
Good work.

File: src/components/editor/Settings.vue (hunk 1 above)
Comment (dakshkhetan) [reply src/components/editor/Settings.vue:48]: So, tabSize = 2, 4 and 8 only?

Comment (hereisnaman) [reply src/components/editor/Settings.vue:48]: yes

Comment (dakshkhetan) [reply src/components/editor/Settings.vue:48]: Okay, working on it.

**Comment by dakshkhetan** [comment]
Updated screenshot:

Screenshot 2020-05-17 at 11 33 20 PM

**Comment by dakshkhetan** [comment]
@hereisnaman I have made the requested changes.

File: src/components/editor/Settings.vue [hunk 2]
```diff
@@ -39,6 +45,7 @@
         },
         fontOptions: ['Lucida Console', 'Anonymous Pro', 'Courier', 'Droid Sans Mono', 'Inconsolata', 'Source Code Pro', 'Ubuntu Mono'],
         sizeOptions: Array(30).fill(0).map((el, ind) => 6 + (2 * ind)),
+        tabSizeOptions: Array(3).fill(0).map((el, ind) => 2 + (2 * ind)).fill(8, 2)
```
Comment (hereisnaman) [inline src/components/editor/Settings.vue:48]: [2,4,8] doesn't hurt.

Comment (dakshkhetan) [reply src/components/editor/Settings.vue:48]: Okay :P 
Working on it.

**Comment by dakshkhetan** [comment]
Done @hereisnaman :)

**Pull request merged**